### PR TITLE
[AT_RU_3] Gallery component

### PR DESCRIPTION
**Pull request by tiwari00ayush**
## Descripitions
1. Added gallery section

## Live Link
https://66c370609e501a04ed6134a5--dapper-speculoos-e087da.netlify.app/

## Screenshots or Recordings
image
image

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] Keyboard accessibility
-   [x] Cross browser testing

### CODE REVIEW

**Review by thisisankur0 — CHANGES_REQUESTED** [review]

File: src/components/Gallery/Gallery.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import {css} from '@emotion/react';
+import {gallery1Image, gallery2Image, gallery3Image, gallery4Image} from '@assets/images';
+import {mq} from '@theme/index';
+
+const styles = {
```
[the remark this answers is not in the document]
Comment (tiwari00ayush) [reply src/components/Gallery/Gallery.tsx:5]: I tried using grid but the image getting distorted somehow that's why i shifted to flex to achieve the same.

File: src/components/Gallery/Gallery.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+            },
+        },
+
+        [mq.SMALL_DESKTOP]: {
+            flex: '1',
+        },
+    }),
+};
+
+export const Gallery = () => {
```
Comment (thisisankur0) [inline src/components/Gallery/Gallery.tsx:98]: Add a tablet view to avoid distortion.

Comment (tiwari00ayush) [reply src/components/Gallery/Gallery.tsx:98]: using this as a tablet will work ?

image

Comment (thisisankur0) [reply src/components/Gallery/Gallery.tsx:98]: Yes

**Review by thisisankur0 — APPROVED** [review]

**Pull request merged**